cpp/ps0000/ps0000_soin_cpp.cpp

```diff
@@ -1,8 +1,9 @@
-#include<iostream>
-#include<conio.h>
+#include <iostream> // 2/2
```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`